apis/s3/v1beta1/lifecycleConfiguration_types.go

```diff
@@ -46,7 +46,7 @@ type LifecycleRule struct {
 	// The Filter is used to identify objects that a Lifecycle Rule applies to.
 	// A Filter must have exactly one of Prefix, Tag, or And specified.
 	// +optional
-	Filter *LifecycleRuleFilter `json:"filter,omitempty"`
+	Filter LifecycleRuleFilter `json:"filter,omitempty"`
```

Comment:
I think we can leave this as `*` because we are creating the struct no matter what in the client

Reply:
Fixed :)